rfc021-platform-policy.md

```diff
@@ -76,38 +74,37 @@ Tier 2 supported platforms are those on which Omnitruck will serve packages, but
 
 Includes any of the add-ons (webui2/manage, push, etc.)
 
-### Supported
+### Foundational
 
 * Ubuntu 12.04LTS, 14.04LTS
 * RHEL 5.x, 6.x, 7.x
 * CentOS 5.x, 6.x, 7.x
 * Oracle Enterprise Linux 5.x, 6.x, 7.x
 
-### Unsupported
+### Other or non-viable
 
 * Any other Linux or UNIX distributions
 * Windows
 
 ## ChefDK
 
-### Supported
+### Foundational
 
 * Windows 7, 8, 8.1
 * Fedora (current non-EOL releases)
 * RHEL 6.x, 7.x
 * Mac OS X 10.9, 10.10, 10.11
 * Ubuntu 12.04, 14.04
 
-ChefDK bundles Chef Client. Therefore, Chef Client is supported, by extension, on the foregoing client platforms, if not already mentioned explicitly in the Chef Client support matrix.
 
-### Unsupported
+###  Other or non-viable
 
 * Windows Vista, XP, 2000
 * Mac OS X < 10.9, anything ppc
 
-## Appendix 1: Guiding Principles for Operating System Version Support
+## Appendix 1: Guiding Principles for Operating System Version Adoption
 
-Once Chef Software, Inc. decides to support an operating system, we will also develop rules to determine under what upstream vendor lifecycle we will continue to support products, and they will be documented in this section. Vendors have various terminology to describe support lifecycles ('standard support', 'extended support', etc.) and it is useful to clarify what those mean in the context of Chef's products.
+Once Chef Software, Inc. decides to adopt an operating system, we will also develop rules to determine under what upstream vendor lifecycle we will continue to build and/or test on our products on that platform, and they will be documented in this section. Vendors have various terminology to describe support lifecycles ('standard support', 'extended support', etc.) and it is useful to clarify what those mean in the context of Chef's products.
```

Comment:
This sentence is kinda awkward, maybe rephrase as something like `...we will also develop rules to determine what upstream vendor lifecycle applies, and will build and/or test our products according to these rules. These rules will be documented in this section.`

I don't know if that muddies up the underlying meaning, but it seems more clear to me.